src/l500/l500-private.h

```diff
@@ -14,10 +14,11 @@ static const int MAX_NUM_OF_DEPTH_RESOLUTIONS = 5;
 
 namespace librealsense
 {
-    const uint16_t L500_RECOVERY_PID    = 0x0b55;
-    const uint16_t L500_PID             = 0x0b0d;
-    const uint16_t L515_PID_PRE_PRQ     = 0x0b3d;
-    const uint16_t L515_PID             = 0x0b64;
+    const uint16_t L500_RECOVERY_PID            = 0x0b55;
+    const uint16_t L500_USB2_RECOVERY_PID_OLD   = 0x0adc; // Units with old DFU_PAYLAOD on USB2 report ds5 PID (RS_USB2_RECOVERY_PID)
```

Comment:
Both `USB2` and `OLD`?

Reply:
`PAYLOAD`

Reply:
They fix the issue on newer units, so at the new units the USB2 report 0x0b55
This is why I called it USB2 + OLD

Reply:
So the NEW USB2 is the same as the USB3 PID?

Reply:
According to my understanding the NEW USB2 is the same as L500_RECOVERY_PID reported on USB3 DFU